pp/integrations/plugin-packs/procedures/hardware-storage-panzura-snmp.md

```diff
@@ -1,49 +1,903 @@
+
+<Tabs groupId="sync">
+<TabItem value="HW-Storage-Panzura-SNMP-custom" label="HW-Storage-Panzura-SNMP-custom">
+
+| Service Alias           | Service Template                                       | Service Description                                           | Discovery |
+|:------------------------|:-------------------------------------------------------|:--------------------------------------------------------------|:---------:|
+| Cpu-Cloud               | HW-Storage-Panzura-Cpu-Cloud-SNMP-custom               | Check usage of cloud controller CPU                           |           |
+| Cpu-Detailed            | HW-Storage-Panzura-Cpu-Detailed-SNMP-custom            | Check the detailed rate of utilization of CPU for the machine |           |
+| Disk-Usage-Cloud-Global | HW-Storage-Panzura-Disk-Usage-Cloud-Global-SNMP-custom | Check cloud disk usages                                       |           |
+| Disk-Usage-Local        | HW-Storage-Panzura-Disk-Usage-Local-SNMP-custom        | Check local disk usage                                        |           |
+| Load                    | HW-Storage-Panzura-Load-SNMP-custom                    | Check the server load average                                 |           |
+| Memory                  | HW-Storage-Panzura-Memory-SNMP-custom                  | Check the rate of the utilization of memory (RAM)             |           |
+| Ratios                  | HW-Storage-Panzura-Ratios-SNMP-custom                  | Check deduplication, compression and save ratios              |           |
+| Swap                    | HW-Storage-Panzura-Swap-SNMP-custom                    | Check virtual memory usage (SWAP)                             |           |
+| Traffic-Global          | HW-Storage-Panzura-Traffic-Global-SNMP-custom          | Check traffic of multiple network interfaces                  |     X     |
+
+> The services listed above are created automatically when the **HW-Storage-Panzura-SNMP-custom** host template is used.
+
+> If **Discovery** is checked, it means a service discovery rule exists for this service template.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias                 | Service Template                                             | Service Description                                                                                                    | Discovery |
+|:------------------------------|:-------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|:---------:|
+| Disk-Io-Generic-Name          | HW-Storage-Panzura-Disk-Io-Generic-Name-SNMP-custom          | Check access disk of the disk. For each check the name of the disk will appear "label" rather than the letter assigned |           |
```

Comment:
```suggestion
| Disk-Io-Generic-Name          | HW-Storage-Panzura-Disk-Io-Generic-Name-SNMP-custom          | Check disk access for the disk. For each check the name of the disk will appear with the "label" rather than the letter assigned |           |
```